pkg/doc/verifiable/credential_ldp_test.go

```diff
@@ -204,13 +204,41 @@ func TestNewCredentialFromLinkedDataProof_JSONLD_Validation(t *testing.T) {
 }
 `
 
-		vcWithLdp, _, err := NewCredential([]byte(vcJSON), vcOptions...)
+		vcWithLdp, _, err := newTestCredential([]byte(vcJSON), vcOptions...)
 		r.Error(err)
 		r.EqualError(err, "JSON-LD doc has different structure after compaction")
 		r.Nil(vcWithLdp)
 	})
 
 	t.Run("VC with different mapped field", func(t *testing.T) {
+		mavennetJSONLD := `
```

Comment:
should replace with our own simple version that tests the changes property name scenario.

Reply:
e.g., a simple context that includes "comments": "schema:text"

Reply:
done!